ntoskrnl/fsrtl/notify.c

```diff
@@ -85,6 +85,7 @@ FsRtlCancelNotify(IN PDEVICE_OBJECT DeviceObject,
     PIO_STACK_LOCATION Stack;
     PNOTIFY_CHANGE NotifyChange;
     PREAL_NOTIFY_SYNC RealNotifySync;
+    BOOLEAN PoolQuotaCharged;
```

Comment:
No, it does not. In a struct, it would add padding, unless you pack it. On the stack, the compiler is free to rearrange variables any way it likes, but it will ensure natural alignment.